### DISCUSSION TITLE
wxFormBuilder as XRC file

### DISCUSSION THREAD

**Discussion by rengel-de** [wxFormBuilder/wxFormBuilder#818] [Q&A]
Hi! I'm new to wxFormBuilder and want to use it to generate UIs for Elixir/Erlang. 
Is there a file **wxFormBuilder.xrc** that contains the definition of wxFormBuilder itself? If so, where?

**Comment by sodevel** [comment]
I'm sorry but i don't understand your question, what do you mean with "definition of wxFormBuilder itself"?

wxFB currently does not support code generation for wxErlang, but there is the outstanding PR #650 that adds such code generation. This PR is quite old and had some issues though, thats why it wasn't merged yet.
